client/my-sites/posts/main.jsx

```diff
@@ -56,56 +60,48 @@ const PostsMain = React.createClass( {
 			return false;
 		}
 
+		if ( ! site.jetpack && this.props.author && this.props.myDraftCount === 0 ) {
+			return false;
+		}
+
 		return true;
 	},
 
 	mostRecentDrafts() {
 		const site = this.props.sites.getSelectedSite();
-		const isLoading = this.props.draftCount !== 0 && this.props.loadingDrafts;
 
 		if ( ! site || ! this.showDrafts() ) {
 			return null;
 		}
 
+		const { draftCount, translate } = this.props;
+		const isLoading = draftCount !== 0 && this.props.loadingDrafts;
+
 		return (
 			<div className="posts__recent-drafts">
 				<QueryPosts
 					siteId={ site.ID }
 					query={ this.props.draftsQuery } />
 				<QueryPostCounts siteId={ site.ID } type="post" />
-				<Card compact className="posts__drafts-header">
-					{ this.translate( 'Latest Drafts' ) }
-					<Button borderless href={ this.props.newPostPath }>
-						<Gridicon icon="plus" />
+				<SectionHeader className="posts__drafts-header" label={ translate( 'Latest Drafts' ) }>
+					<Button compact href={ this.props.newPostPath }>
+						{ translate( 'Start New' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Start a new post' )` *ES Score: 6.71*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/use-new-cpt-post-component-for-drafts)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).